addons/sale_loyalty/tests/test_program_numbers.py

```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
```

Comment:
It's correct, but this line will be taken off in master (already merged), so if you want to avoid a conflict, I don't mind letting it for now.